packages/terra-table/docs/TableHeader.md

```diff
@@ -0,0 +1,33 @@
+# Terra Table Header
+
+The Terra Table Header is a sub component passed to the table as a prop, rather than children. The header provides the layout for header cells.
```

Comment:
passed to the table ~as a prop,~ through Table's `header` prop,

Reply:
Updated.